### DISCUSSION TITLE
Can `QueueEvents` listen across different types of workers?

### DISCUSSION THREAD

**Discussion by amitav13** [taskforcesh/bullmq#449] [Ideas]
Is it possible to have one `QueueEvents` instance that logs events for ALL workers, not just the ones filtered by the `name`? My use case is just general logging like:
```js
const queueEvents = new QueueEvents("*"); //All events across all workers
queueEvents.on("active", (jobId) => {
  logger.info(`Job ${jobId} is active`);
});
```

**Comment by roggervalf** [comment]
No currently, as each instance is listening a dedicated stream
